src/main/resources/application.yaml

```diff
@@ -186,3 +190,12 @@ springdoc:
     enabled: true
     operationsSorter: method
   writer-with-order-by-keys: true
+
+professional:
+  refdata:
+    pageSize: ${PROFESSIONAL_REFDATA_PAGE_SIZE:100}
+  role:
+    mapping:
+      scheduling:
+        enabled: ${PROFESSIONAL_SCHEDULING_ENABLED:true}
+        cron: ${PROFESSIONAL_ROLE_MAPPING_CRON:*/5 * * * * *}
```

Comment:
can you put your cron behind its own environment variable. see application.yaml on GA-55 as an example https://github.com/hmcts/am-org-role-mapping-service/pull/1743/files#diff-b4c0421bcb3280a8e738550c830dba197057c515bcdd9579a52cbfe9adca2061